app/components/layer-advanced/template.hbs

```diff
@@ -0,0 +1,4 @@
+{{yield}}
```

Comment:
You can remove the yields, I don't think we will inject anything into these components.